charm/kafka/reactive/kafka.py

```diff
@@ -154,14 +80,10 @@ def configure_kafka_zookeepers(zk):
             data_changed('kafka.storage.log_dir', log_dir)))):
         return
 
-    if len(zks) < 1:
```

Comment:
Please revert.

Reply:
@cmars if there are no zk nodes it would go back to waiting for zookeeper relation message, so it’s a bit useless.